install/bruteforce.sh

```diff
@@ -0,0 +1,3 @@
+sudo apt-get install -y python-pip python-dev
+sudo apt-get install -y libatlas-dev libatlas3gf-base
+sudo apt-get install -y python-numpy
```

Comment:
@erikbern I'm not terribly sure how the debian-packaged NumPy plays with BLAS... can you check that BLAS is being picked up by NumPy (=dot calls are fast)?